transform/mattermost-analytics/models/intermediate/sales/hightouch/int_onprem_subscriptions.sql

```diff
@@ -0,0 +1,66 @@
+{{config({
+    "materialized": 'table',
+    "unique_key":"invoice_id",
+  })
+}}
+
+
+WITH subscriptions AS (
+select s.subscription_id
+    , s.purchase_order_number
+    , s.stripe_customer_id 
+    , s.license_id
+    , s.edition
+    , s.product
+    , p.sku
+    FROM {{ ref('stg_stripe__subscriptions') }}  s 
+    JOIN {{ ref('stg_stripe__products') }} p ON s.product = p.product_id
```

Comment:
3/5 Think that `product`  in `stg_stripe__products` needs to be renamed to `product_id`.

Reply:
Updated to product_id